### DISCUSSION TITLE
is it worth putting?

### DISCUSSION THREAD

**Discussion by 192IP** [PilkDrinker/PilkDots#10] [General]
I'm thinking of installing this dot, but I'm worried about how much RAM it eats

**Comment by PilkDrinker** [comment]
Here's how much memory the rice itself use on my Machine.

hyprland:     460 - 470 MB
waybar:        95   MB (Constant)
rofi:               35   MB (when opened)
swaync:        50   MB (Constant)
swww-deamon:         39   MB
swww (When switching)        39 - 80 MB 

Around **645 MB** when idle.
And about **770 MB** when swaync, rofi and swww are all active at once.

**Reply by 192IP** [reply]
thank u
